src/osha/oira/client/browser/client.py

```diff
@@ -40,13 +41,9 @@ def _get_entry(self, list_id, title):
             "text": title,
         }
 
-    @property
-    def results(self):
-        """List of "mailing list" path/names.
-
-        The format fits pat-autosuggest. There is a special label for
-        the "all" list.
-        """
+    def validate_ticket(self):
```

Comment:
This might be better called `user_id`

Reply:
But that does not communicate what it does.

Reply:
If you think it should be called `user_id` because it returns a user id, then I'd rather modify it to not return the user id and only validate the ticket, and then just get the user id from the request separately in `results()`. That way it's clearer from the name what the method does.

Reply:
Indeed they should be better separated